packages/tailwindcss/tailwindcss.3.0.23/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+synopsis: "TailwindCSS prebuild command line on opam"
+description: """TailwindCSS prebuild command line on opam."""
+maintainer: ["Thibaut Mattio"]
+authors: ["Thibaut Mattio"]
+license: "ISC"
+homepage: "https://github.com/tmattio/opam-tailwindcss"
+doc: "https://tmattio.github.io/opam-tailwindcss/"
+bug-reports: "https://github.com/tmattio/opam-tailwindcss/issues"
+dev-repo: "git+https://github.com/tmattio/opam-tailwindcss.git"
+install: [ 
+  "cp" 
```

Comment:
Same problem as `esbuild`. I think a conf-package would be more suitable or using a conf- package that calls npm or system package managers